apollo-router/tests/integration/telemetry/fixtures/trace_id_via_header.router.yaml

```diff
@@ -1,9 +1,6 @@
 telemetry:
-
```

Comment:
Was using this test as a basis for my new test and cleaned up some stuff in a process